examples/complete-integration/app/server.ts

```diff
@@ -0,0 +1,85 @@
+import express from "express";
+import bodyParser from "body-parser";
+import {
+  configureTestBox,
+  TestBoxUseCaseRequest,
+  TestBoxTrial,
+  TestBoxTrialRequest,
+  TestboxConfigFramework,
+} from "@testboxlab/node-sdk";
+
+configureTestBox({
+  productId: "my-slugified-product-name",
+  framework: TestboxConfigFramework.EXPRESS,
+});
+
+const app = express();
+app.use(bodyParser.json());
+
+app.post("/api/testbox/trial", async (req, res) => {
+  const trialRequest = await TestBoxTrialRequest.fromExpressRequest(req);
```

Comment:
Don't need to change -- but since we have configured TestBox to be EXPRESS above, we would want this to just be TestBoxTrialRequest.fromRequest(request) or even just TestBoxTrialRequest(req)